deploy/openshift/templates/che-workspace-service-account.yaml

```diff
@@ -76,3 +76,7 @@ parameters:
   displayName: Service account namespace
   description: Namespace where service account should be created
   required: true
+labels:
+  app: che
+  component: che
+  template: che-workspace-service-account
```

Comment:
Why do we need template here?

Reply:
We don't really need it but all other OpenShift templates have it, so I've added to this template as well.